brave/src/main/java/brave/propagation/B3Propagation.java

```diff
@@ -115,10 +116,14 @@ public FactoryBuilder injectFormats(Span.Kind kind, Format format1, Format forma
    * decision to the receiver of this header).
    */
   static final String SAMPLED_NAME = "X-B3-Sampled";
+  static final String SAMPLED_MALFORMED =
+    "Invalid input: expected 0 or 1 for " + SAMPLED_NAME + ": {0}";
```

Comment:
Perhaps `: '{0}'` with quotes - IIUC empty string will also log so if it just logs as empty it's tricky to read the log. But either's fine.

Reply:
good idea.. in this case I'll put the whole thing in single quotes as was aiming for header format.

Ex. `X-B3-Sampled: ` in the case of empty string

Reply:
but then again, that would still have the same problem.. yeah single quoting it